CHANGES

```diff
@@ -79,6 +89,16 @@
      (CVE-2019-1543)
      [Matt Caswell]
 
+  *) Add DEVRANDOM_WAIT feature for Linux systems
+
+     On older Linux systems where the getrandom() system call is not available,
+     OpenSSL normally uses the /dev/urandom device for seeding its CSPRNG.
+     Contrary to getrandom(), the /dev/urandom device will not block during
+     early boot when the kernel CSPRNG has not been initially seeded yet.
```

Comment:
"initially seeded yet" is a tautology - either "initially seeded" or "seeded yet".

Reply:
I'll go for "has not been seeded yet."